rest/nft.go

```diff
@@ -0,0 +1,82 @@
+package rest
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/likecoin/likecoin-chain-tx-indexer/db"
+)
+
+func handleNftClass(c *gin.Context) {
+	var q db.QueryClassRequest
+
+	if err := c.ShouldBindQuery(&q); err != nil {
+		c.AbortWithStatusJSON(400, gin.H{"error": err.Error()})
+		return
+	}
+
+	conn := getConn(c)
+	res, err := db.GetClasses(conn, q)
```

Comment:
Should we do simple check on these querys if required qs exists and valid?  e.g. iscn_id_prefix for this query

Reply:
I have add `binding: "required"` in the structure tag in db/types.go, so gin would do this job.